configure.ac

```diff
@@ -32,15 +32,12 @@ AC_LANG(C)
 
 LT_INIT
 
-source $srcdir/versions.txt
-
 # Check for macros
 AC_DEFUN_ONCE([MACRO_CHECK],
 [
-	ifdef([$1], true, [AC_FATAL([Please install autoconf-archive version ${autoconf_archive_version}],)])
+	ifdef([$1], true, [AC_FATAL([Please install autoconf-archive],)])
 ])
 
-MACRO_CHECK([AX_VALGRIND_DFLT])
```

Comment:
This macro will check for AX_VALGRIND_CHECK AX_CODE_COVERAGE that  are provided by autoconf archive.

I wonder if we can modify MACRO_CHECK, to not fail de execution but let us know if the AX_VALGRIND_CHECK and others are defined.

If they are defined the m4 files exist and we could use valgrind and code coverage, otherwise we continue with the execution without execute valgrind test.